src/edu/stanford/dlss/was/SettingsLoadException.java

```diff
@@ -0,0 +1,14 @@
+package edu.stanford.dlss.was;
+
+/**
+ * A small class to represent exceptions in general when loading the app settigns.
+ *
+ * Prevents callers that ask for settings initialization from having to know
+ * too much detail about the more granular exceptions that might get thrown
+ * when reading properties, parsing CLI args, etc.
+ */
```

Comment:
ah, sorry, i kept the wordier version but put it in javadoc format.  i figured the javadoc part was the thing you were really getting at.  anyway, made it more concise (though slightly wordier than this suggestion).